lib/signaling/v2/room.js

```diff
@@ -220,6 +230,20 @@ class RoomV2 extends RoomSignaling {
     return this;
   }
 
+  /**
+   * @private
+   */
+  _createRemoteParticipant(participantState) {
+    const { _RemoteParticipantSignaling: RemoteParticipantV2 } = this;
```

Comment:
_RemoteParticipantSignaling can now be RemoteParticipantV3 or RemoteParticipantV2 right? should this be:
```suggestion
    const { _RemoteParticipantSignaling: RemoteParticipantSignalingClassType } = this;
```

Reply:
never mind this. I understand that it will always be `RemoteParticipantV2` as its v2 version. Ignore my suggestions Looks good, its easier to search this way.